.github/workflows/_bazel-build-test.yml

```diff
@@ -11,13 +11,25 @@ on:
         required: true
         type: string
         description: Name of the base docker image to build with.
+      cuda-version:
+        required: false
+        type: string
+        description: What CUDA version to build with, "cpu" for none.
+        default: "default"
       sync-tag:
         required: false
         type: string
         default: ""
         description: |
           If this is set, our linter will use this to make sure that every other
           job with the same `sync-tag` is identical.
+      test-matrix:
+        required: false
+        type: string
+        description: |
+          An option JSON description of what test configs to run later on. This
+          is moved here from the Linux test workflow so that we can apply filter
```

Comment:
nit: this second sentence can be updated here: just say that we provide all test configs at build time to be able to only do the necessary work.